src/index.ts

```diff
@@ -37,10 +41,157 @@ function gistIdFromTwiddleUrl(twiddleUrl: string): string {
   }
 }
 
-export default function twiddleEject(twiddleUrl: string, projectName: string, _options: {} = {}) {
+/**
+ * Given a path, generate single or multi level directory.
+ * @param path
+ */
+function generateDir(path: string) {
```

Comment:
http://adilapapaya.com/docs/fs-extra/#mkdirsdircallback

is a more robust thing to use here

```ts
import * as fs from 'fs-extra';

fs.mkdirsSync(path, { recursive: true });
```